include/lorawan.h

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
We are trying to get rid of various includes from `include/` so this is not good place for lorawan.h. You could place the include file into `include/net/` directory as there are other radio API includes there already.

Reply:
Okay. We had this discussion for previous LoRa support also. But we settled with `include/drivers/lora.h`. In this case, since we don't have `include/subsys` I think it can live under `include/net`

Reply:
@Mani-Sadhasivam If lorawan.h is moving to /net, should the sample application fall under samples/net?